ecs-cli/modules/cli/compose/entity/service/service_helper.go

```diff
@@ -41,16 +95,26 @@ func waitForServiceTasks(service *Service, ecsServiceName string) error {
 			"desiredCount": desiredCount,
 			"runningCount": runningCount,
 		}
+
+		// Log information only if things have changed
+		// running count has changed
```

Comment:
Perhaps combine these two comment lines to "Log information only if runningCount has changed"?